Engine/EditorUI/Panel/PopupsPanel/PanelPopupTextureSelector.h

```diff
@@ -0,0 +1,33 @@
+#ifndef _PANELPOPUPTEXTURESELECTOR_H_
+#define _PANELPOPUPTEXTURESELECTOR_H_
+
+#include "EditorUI/Panel/Panel.h"
+#include "Module/ModuleFileSystem.h"
+
+#include <string>
+
+class PanelPopupTextureSelector : public Panel
+{
+public:
+	PanelPopupTextureSelector();
+	~PanelPopupTextureSelector() = default;
+
+	void Render() override;
+
+private:
+	void ShowTextureIcon(File* file);
+	void ProcessTextureMouseInput(File * file);
+	void ChangeSelectedBillboardTexture(std::string path) const;
```

Comment:
sent a reference or a pointer

Reply:
i just find out that this panel is not used anyware, I didn't created, but I'm going to delete it